subsys/bluetooth/mesh/Kconfig

```diff
@@ -380,6 +380,17 @@ endchoice
 
 if BT_MESH_ADV_EXT
 
+config BT_MESH_ADV_MAIN_RELAY_ENABLED
+	bool "Advertiser main support relay messages"
+	default y if !BT_MESH_FRIEND && !BT_MESH_LOW_POWER
+	depends on BT_MESH_RELAY
```

Comment:
Should it also depend on CONFIG_BT_MESH_RELAY_ADV_SETS? If CONFIG_BT_MESH_RELAY_ADV_SETS is zero, shouldn't the main adv set be used for relaying messages?